### ISSUE TITLE
Many codebase versions with same DB

### ISSUE THREAD

**Issue by jogo-devs** [keystonejs/keystone#9364]
1. Create a project Keystonejs
2. Create a DB in PostgreSQL for development mode
2. Upload or shared your project with other developer (Developer 2) include DB shared
3. (Developer 2) added a new field in any model and start server
4. (Developer 1) Start server without new update of (developer 2)
5. Keystonejs is not able to lift and function without knocking over your partner's work

I should be able to run different versions using the same database, it's fine to create this Data Cleanup but it should be some manual script not something automatic, this is very sad to work with other developers because it's a drag, all other Headless CMS options do this without problems, why not Keystonejs?

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/87f2d17a-ff5d-4373-a8cf-834b1e66148b">

Any Nodejs version

**Comment by marekryb** [comment]
https://keystonejs.com/docs/guides/database-migration#skipping-dev-db-push 
TLDR: `keystone dev --no-db-push`

I would recommend using migrations instead. You will need them in production anyway.

**Comment by jogo-devs** [comment]
However, why should I need this in either prod or dev, when other CMS do it automatically without any problem?

**Comment by jogo-devs** [comment]
This also does not solve anything, because it starts giving errors everywhere because the new fields do not exist in the DB.

**Comment by rivaslive** [comment]
Same here!

**Comment by dcousens** [comment]
I don't really understand what you're asking here, if your databases aren't aligned, you can expect problems.
